php/php.code.analysis/src/org/netbeans/modules/php/analysis/commands/CodeSniffer.java

```diff
@@ -237,8 +238,11 @@ public void run() {
     private List<String> getParameters(String standard, FileObject file, boolean noRecursion) {
         Charset encoding = FileEncodingQuery.getEncoding(file);
         List<String> params = new ArrayList<>();
-        // #270987 use --standard
-        params.add(String.format(STANDARD_PARAM, standard));
+        // NETBEANS-3243 the path of Code Sniffer may have --standard parameter
+        if (!codeSnifferPath.contains(STANDARD_PARAM)) {
```

Comment:
Nitpick: A bit more precise/safe would be to check for `--standard=` to avoid confusion with (possible future?) parameters like `--standard-log-file` etc.

Reply:
Yes, I'll fix it.

Reply:
Fixed: https://github.com/apache/netbeans/compare/7fdb5a05aa6b55d6f2f65731b3e192bdcd24897f..2f983b95ea6878c8d027a65caf75ce431c9b012e

Reply:
Nice, thanks. BTW I am not sure whether the command cannot be `[...] --standard <something> [...]` (I mean using ` ` instead of `=`).

Reply:
Maybe, you are right. We should check both cases, just in case. Will fix it.

Reply:
Fixed.